### PR TITLE
fix: count "yes" and "no" as part of getVariant

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR fixes a behavior issue related to counting metrics: when counting variant metrics, we should also count "yes" and "no" / enabled and disabled.

This PR changes one of the existing tests to align with this behavior and adds an extra test to specifically about this.

### CODE REVIEW

File: src/main/kotlin/io/getunleash/UnleashClient.kt [hunk 1]
```diff
@@ -88,7 +88,10 @@ class UnleashClient(
     }
 
     override fun getVariant(toggleName: String): Variant {
-        val variant = refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
+        val variant =
+                if (isEnabled(toggleName))
+                        refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
+                else disabledVariant
```
Comment (chriswk) [inline src/main/kotlin/io/getunleash/UnleashClient.kt:94]:
```suggestion
                if (isEnabled(toggleName)) {
                        refreshPolicy.readToggleCache()[toggleName]?.variant ?: disabledVariant
} else {
  disabledVariant
}
```
I know it's allowed to not do it, but if/else should always have braces. I was pretty sure we had a lint for this.

Comment (chriswk) [reply src/main/kotlin/io/getunleash/UnleashClient.kt:94]: Also, I see I broke the formatting here. The suggestion box didn't show properly

**Review by chriswk — APPROVED** [review]
Nice. Thanks for doing this.

File: src/main/kotlin/io/getunleash/UnleashClient.kt (hunk 1 above)
Comment (thomasheartman) [reply src/main/kotlin/io/getunleash/UnleashClient.kt]: Resolved in 126fbd07217db581e946a463988459fd4a1baaa8

**Pull request merged**